src/components/ProvinceList.jsx

```diff
@@ -12,34 +12,48 @@ const linkStyle = {
 };
 
 export default function ProvinceList() {
+  const [result, setResult] = useState('');
   const { provinces, isLoading } = useSelector((store) => store.provinces);
   const dispatch = useDispatch();
+
+  const filtered = provinces.filter((el) => el.name.toLowerCase().includes(result));
+
```

Comment:
- [OPTIONAL] I noticed that the search functionality only works when the user's input is in lowercase. When it is in uppercase, it doesn't work. I would advise you to convert the user's input to lowercase even if the user is typing in uppercase. 👍 😎 

#### Hint:

```JavaScript
const filtered = provinces.filter((el) => el.name.toLowerCase().includes(result.toLowerCase()));
```

Reply:
Hi @Gambit142 .
Thanks for your suggestion. I actually, noticed that issue and applied it. I guess, I might have pushed it after your review. Here is the [commit](https://github.com/M-Anwar-Hussaini/Weather-Forecast/pull/1/commits/6976282d813a2dda5bc825446a4e5f1a0a33596b).